### DISCUSSION TITLE
지난 기출 문제 중 2^String의 의미 (Meaning of "2^String")

### DISCUSSION THREAD

**Discussion by 33577** [prosyslab-classroom/cs524-program-analysis#445] [Q&A]
### Your Name

전종욱

### Question

안녕하세요 
2023 가을 IS593 기말 기출 중 3번 문제에 "2^String"이란 표현이 있습니다.
"2^String"은 무슨 뜻인가요? 맥락상 두개의 "String"을 뜻하는 "String X String"인가 싶은데 그럼 "String^2"라고 해야하는 것 아닌가 궁금합니다. 
캡쳐하여 인용하고 싶으나, KLMS에 올려주신 것이 공개된 장소에 게시를 원하시지 않는 것 같아 불편하더라도 캡쳐를 생략했습니다. 
감사합니다.

What is the meaning of "2^String" in the 3rd problem of 2023 Fall Final?

**Comment by KihongHeo** [comment]
$2^X$는 X의 멱집합을 뜻합니다.

**Reply by 33577** [reply]
아 설명해주셔서 감사합니다!
